server/koppsApi.js

```diff
@@ -123,8 +122,30 @@ async function getCourseRoundTerms(courseCode) {
   }
 }
 
+async function getApplicationFromLadokID(ladokRoundId) {
+  const { client } = api.koppsApi
+  const uri = `${config.koppsApi.proxyBasePath}courses/offerings/roundnumber?ladokuid=${ladokRoundId}`
+  log.info('Trying fetch courses application by', { ladokId: ladokRoundId, uri, config: config.koppsApi })
+  try {
+    const res = await client.getAsync({ uri, useCache: true })
+    if (res.body) {
+      const { body } = res
```

Comment:
Destructure body above in await as I described above, it is the same as written code just in one line `const { body } = await client.getAsync({ uri, useCache: true })`

Reply:
done